README.md

```diff
@@ -92,13 +92,33 @@ To view runtime log output:
 $ sudo journalctl -t cc-runtime
 ```
 
-### Enabling debug for various components
+To view shim log output:
+
+```bash
+$ sudo journalctl -t cc-shim
+```
+
+To view proxy log output:
+
+```bash
+$ sudo journalctl -t cc-proxy
+```
+
+Note that the proxy log entries also include output from the agent (`cc-agent`) and the
+hypervisor (which includes the guest kernel boot-time messages).
+
+## Debugging
 
-The runtime, the shim (`cc-shim`), and the hypervisor all have separate debug
-options in the [configuration file](#Configuration).
+The runtime, the shim (`cc-shim`), the proxy (`cc-proxy`) and the hypervisor all have separate `enable_debug=` debug
+options in the [configuration file](#Configuration). By default, all these
+debug options are disabled. Look at the comments in the installed
+configuration file for further details.
 
-The proxy (`cc-proxy`) has a command-line option to enable debug output. See
-the [proxy documentation](https://github.com/clearcontainers/proxy#debugging) for further details.
+If you wish to enable debug for all components, assuming a standard configuration file path, run:
+
+```bash
+$ sudo sed -i -e 's/^#\(enable_debug\).*=.*$/\1 = true/g' /usr/share/defaults/clear-containers/configuration.toml
+```
```

Comment:
Looking much better, thanks @jodh-intel 
Sorry to be a mild pain, but one more suggestion - do you think we should also link from here to the more detailed/specific debug docs we now have as well?
https://github.com/clearcontainers/runtime/blob/master/docs/debug-agent.md
https://github.com/clearcontainers/runtime/blob/master/docs/debug-kernel.md

Reply:
Done. Note that those 2 docs had rotted a bit so I've had to update them. In fact, I'm wondering if we want to merge them (maybe into the `README` itself) since as you can see, they are now very similar.

Reply:
Thanks @jodh-intel 
lgtm